lib/ansible/module_utils/cnos.py

```diff
@@ -3179,7 +3179,10 @@ def waitForDeviceResponse(command, prompt, timeout, obj):
             if(gotit != -1):
                 flag = True
         except:
-            retVal = retVal + "\n Error-101"
+            if(prompt != "(yes/no)?" ):
```

Comment:
Note: conditionals should usually be written like this in python:
``` python
if prompt != "(yes/no)?":
```

leaving out the parenthesis as if and elif are not function calls.  The one time people break this general rule is when the conditional is long enough to span more than one line.  Then they write the conditional like this:
``` python
if (long_variable_name != "other_long_condtion_to_match"
    or second_variable in ('rest', 'of', 'condition')):
```
with a space between the if and the opening parenthesis.  (No need to change all of the existing code but a note to fix this up as you make changes).